package.json

```diff
@@ -24,13 +24,14 @@
   "activationEvents": [
     "workspaceContains:jest.config.js",
     "workspaceContains:jest.json",
-    "workspaceContains:node_modules/.bin/jest",
-    "workspaceContains:node_modules/react-scripts/node_modules/.bin/jest",
+    "workspaceContains:node_modules/.bin/jest?(.cmd)",
+    "workspaceContains:node_modules/react-scripts/node_modules/.bin/jest?(.cmd)",
     "workspaceContains:node_modules/react-native-scripts",
     "onCommand:io.orta.jest.start"
   ],
   "main": "./out/extension",
   "icon": "images/vscode-jest.png",
+  "extensionKind": "workspace",
```

Comment:
Based on VS Code documentation, I believe this is the correct configuration. This extension should work at the workspace level, not at the UI level. So, if you're using any of the remote extensions (like WSL or SSH), this extension should be installed on that remote system to work properly.

For instance, using VS Code for Windows and opening a project within WSL (with all dependencies installed through WSL) would result in having a Jest binary for Linux.  If I'm not mistaken and if `extensionKind` was set to `ui` (or not set at all) this extension would run under Windows (and not WSL as it should) and all changes in this PR would try to find a `jest.cmd` binary, which will not exist.

> I guess we then also have to check whether that works with the VS Code Server running on a remote machine.

I don't think this setting should have any effect on that; it should only affect projects opened with one of the remote extensions.

Reply:
It is the correct setting for sure, I was just wondering whether it could introduce some UI related issues, since we are also doing some UI stuff. But as far as I have understood it, I don't think so.